code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -216,12 +216,13 @@
 /obj/item/melee/energy/sword
 	color
 	name = "energy sword"
-	desc = "May the force be within you."
+	desc = "An energy sword. Quite rare, very dangerous."
 	desc_antag = "The energy sword is a very strong melee weapon, capable of severing limbs easily, if they are targeted.  It can also has a chance \
 	to block projectiles and melee attacks while it is on and being held.  The sword can be toggled on or off by using it in your hand.  While it is off, \
 	it can be concealed in your pocket or bag."
 	icon_state = "sword0"
 	active_force = 30
+	armor_penetration = 25
```

Comment:
note that this is an increase of 5 from the baseline of 20. intentional?

Reply:
Technically, yes. I wasn't right about the amount of AP the e-sword has atm, but nonetheless 25 is a better value than 20. I want the e-sword to be threatening even to someone in riot/heavy armor, and 25 achieves that without being overwhelming. 20 does substantially less damage.

Reply:
👍